[{"TimeUTC":"2025-09-23 20:53:39","timestampInMs":1758660819260,"requestPath":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app/favicon.png","requestMethod":"GET","requestQueryString":"","responseStatusCode":-1,"requestId":"lcd8g-1758660817512-0bf4bf9ab2c6","requestUserAgent":"vercel-favicon/1.0","level":"error","environment":"production","branch":"main","vercelCache":"","type":"lambda","function":"/mcp_server.py","host":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app","deploymentDomain":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app","deploymentId":"dpl_AC5hyoygEwFaK8DBZRvDxtmCqmtR","durationMs":-1,"region":"iad1","maxMemoryUsed":-1,"memorySize":-1,"message":"Traceback (most recent call last):\nFile \"/var/task/vc__handler__python.py\", line 38, in <module>\n__vc_spec.loader.exec_module(__vc_module)\nFile \"<frozen importlib._bootstrap_external>\", line 999, in exec_module\nFile \"<frozen importlib._bootstrap>\", line 488, in _call_with_frames_removed\nFile \"/var/task/mcp_server.py\", line 20, in <module>\nfrom mcp.server.auth.provider import TokenVerifier, AccessToken\nImportError: cannot import name 'TokenVerifier' from 'mcp.server.auth.provider' (/var/task/_vendor/mcp/server/auth/provider.py)\nPython process exited with exit status: 1. The logs above can help with debugging the issue.","projectId":"prj_mzlT1PT31kIcvA2XkLZ4NwvwbESq","traceId":"","sessionId":"","invocationId":"01K5W5MWMFKY9KCMJK0WR8YJ2Q","instanceId":"","concurrency":-1},{"TimeUTC":"2025-09-23 20:53:37","timestampInMs":1758660817551,"requestPath":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app/favicon.png","requestMethod":"GET","requestQueryString":"","responseStatusCode":500,"requestId":"lcd8g-1758660817512-0bf4bf9ab2c6","requestUserAgent":"vercel-favicon/1.0","level":"error","environment":"production","branch":"main","vercelCache":"","type":"lambda","function":"/mcp_server.py","host":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app","deploymentDomain":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app","deploymentId":"dpl_AC5hyoygEwFaK8DBZRvDxtmCqmtR","durationMs":1689,"region":"iad1","maxMemoryUsed":100,"memorySize":2048,"message":"","projectId":"prj_mzlT1PT31kIcvA2XkLZ4NwvwbESq","traceId":"","sessionId":"","invocationId":"01K5W5MWMFKY9KCMJK0WR8YJ2Q","instanceId":"","concurrency":1},{"TimeUTC":"2025-09-23 20:53:35","timestampInMs":1758660815430,"requestPath":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app/favicon.png","requestMethod":"GET","requestQueryString":"","responseStatusCode":-1,"requestId":"mzztn-1758660813739-c5693b09dc38","requestUserAgent":"vercel-favicon/1.0","level":"error","environment":"production","branch":"main","vercelCache":"","type":"lambda","function":"/mcp_server.py","host":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app","deploymentDomain":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app","deploymentId":"dpl_AC5hyoygEwFaK8DBZRvDxtmCqmtR","durationMs":-1,"region":"iad1","maxMemoryUsed":-1,"memorySize":-1,"message":"Traceback (most recent call last):\nFile \"/var/task/vc__handler__python.py\", line 38, in <module>\n__vc_spec.loader.exec_module(__vc_module)\nFile \"<frozen importlib._bootstrap_external>\", line 999, in exec_module\nFile \"<frozen importlib._bootstrap>\", line 488, in _call_with_frames_removed\nFile \"/var/task/mcp_server.py\", line 20, in <module>\nfrom mcp.server.auth.provider import TokenVerifier, AccessToken\nImportError: cannot import name 'TokenVerifier' from 'mcp.server.auth.provider' (/var/task/_vendor/mcp/server/auth/provider.py)\nPython process exited with exit status: 1. The logs above can help with debugging the issue.","projectId":"prj_mzlT1PT31kIcvA2XkLZ4NwvwbESq","traceId":"","sessionId":"","invocationId":"01K5W5MRYRJS3J4Z9E384BVWQ3","instanceId":"","concurrency":-1},{"TimeUTC":"2025-09-23 20:53:33","timestampInMs":1758660813784,"requestPath":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app/favicon.png","requestMethod":"GET","requestQueryString":"","responseStatusCode":500,"requestId":"mzztn-1758660813739-c5693b09dc38","requestUserAgent":"vercel-favicon/1.0","level":"error","environment":"production","branch":"main","vercelCache":"","type":"lambda","function":"/mcp_server.py","host":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app","deploymentDomain":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app","deploymentId":"dpl_AC5hyoygEwFaK8DBZRvDxtmCqmtR","durationMs":1629,"region":"iad1","maxMemoryUsed":100,"memorySize":2048,"message":"","projectId":"prj_mzlT1PT31kIcvA2XkLZ4NwvwbESq","traceId":"","sessionId":"","invocationId":"01K5W5MRYRJS3J4Z9E384BVWQ3","instanceId":"","concurrency":1},{"TimeUTC":"2025-09-23 20:53:32","timestampInMs":1758660812688,"requestPath":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app/favicon.png","requestMethod":"GET","requestQueryString":"","responseStatusCode":-1,"requestId":"n9t9s-1758660810959-b2af4ebdf962","requestUserAgent":"vercel-favicon/1.0","level":"error","environment":"production","branch":"main","vercelCache":"","type":"lambda","function":"/mcp_server.py","host":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app","deploymentDomain":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app","deploymentId":"dpl_AC5hyoygEwFaK8DBZRvDxtmCqmtR","durationMs":-1,"region":"iad1","maxMemoryUsed":-1,"memorySize":-1,"message":"Traceback (most recent call last):\nFile \"/var/task/vc__handler__python.py\", line 38, in <module>\n__vc_spec.loader.exec_module(__vc_module)\nFile \"<frozen importlib._bootstrap_external>\", line 999, in exec_module\nFile \"<frozen importlib._bootstrap>\", line 488, in _call_with_frames_removed\nFile \"/var/task/mcp_server.py\", line 20, in <module>\nfrom mcp.server.auth.provider import TokenVerifier, AccessToken\nImportError: cannot import name 'TokenVerifier' from 'mcp.server.auth.provider' (/var/task/_vendor/mcp/server/auth/provider.py)\nPython process exited with exit status: 1. The logs above can help with debugging the issue.","projectId":"prj_mzlT1PT31kIcvA2XkLZ4NwvwbESq","traceId":"","sessionId":"","invocationId":"01K5W5MP7THB8Z84YXXAGC2BEH","instanceId":"","concurrency":-1},{"TimeUTC":"2025-09-23 20:53:31","timestampInMs":1758660811002,"requestPath":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app/favicon.png","requestMethod":"GET","requestQueryString":"","responseStatusCode":500,"requestId":"n9t9s-1758660810959-b2af4ebdf962","requestUserAgent":"vercel-favicon/1.0","level":"error","environment":"production","branch":"main","vercelCache":"","type":"lambda","function":"/mcp_server.py","host":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app","deploymentDomain":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app","deploymentId":"dpl_AC5hyoygEwFaK8DBZRvDxtmCqmtR","durationMs":1669,"region":"iad1","maxMemoryUsed":100,"memorySize":2048,"message":"","projectId":"prj_mzlT1PT31kIcvA2XkLZ4NwvwbESq","traceId":"","sessionId":"","invocationId":"01K5W5MP7THB8Z84YXXAGC2BEH","instanceId":"","concurrency":1},{"TimeUTC":"2025-09-23 20:53:30","timestampInMs":1758660810919,"requestPath":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app/favicon.ico","requestMethod":"GET","requestQueryString":"","responseStatusCode":-1,"requestId":"ppcng-1758660809091-26a1c8dd3b42","requestUserAgent":"vercel-favicon/1.0","level":"error","environment":"production","branch":"main","vercelCache":"","type":"lambda","function":"/mcp_server.py","host":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app","deploymentDomain":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app","deploymentId":"dpl_AC5hyoygEwFaK8DBZRvDxtmCqmtR","durationMs":-1,"region":"iad1","maxMemoryUsed":-1,"memorySize":-1,"message":"Traceback (most recent call last):\nFile \"/var/task/vc__handler__python.py\", line 38, in <module>\n__vc_spec.loader.exec_module(__vc_module)\nFile \"<frozen importlib._bootstrap_external>\", line 999, in exec_module\nFile \"<frozen importlib._bootstrap>\", line 488, in _call_with_frames_removed\nFile \"/var/task/mcp_server.py\", line 20, in <module>\nfrom mcp.server.auth.provider import TokenVerifier, AccessToken\nImportError: cannot import name 'TokenVerifier' from 'mcp.server.auth.provider' (/var/task/_vendor/mcp/server/auth/provider.py)\nPython process exited with exit status: 1. The logs above can help with debugging the issue.","projectId":"prj_mzlT1PT31kIcvA2XkLZ4NwvwbESq","traceId":"","sessionId":"","invocationId":"01K5W5MMG4WAX33FYV623EPPQ4","instanceId":"","concurrency":-1},{"TimeUTC":"2025-09-23 20:53:29","timestampInMs":1758660809220,"requestPath":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app/favicon.ico","requestMethod":"GET","requestQueryString":"","responseStatusCode":500,"requestId":"ppcng-1758660809091-26a1c8dd3b42","requestUserAgent":"vercel-favicon/1.0","level":"error","environment":"production","branch":"main","vercelCache":"","type":"lambda","function":"/mcp_server.py","host":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app","deploymentDomain":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app","deploymentId":"dpl_AC5hyoygEwFaK8DBZRvDxtmCqmtR","durationMs":1682,"region":"iad1","maxMemoryUsed":100,"memorySize":2048,"message":"","projectId":"prj_mzlT1PT31kIcvA2XkLZ4NwvwbESq","traceId":"","sessionId":"","invocationId":"01K5W5MMG4WAX33FYV623EPPQ4","instanceId":"","concurrency":1},{"TimeUTC":"2025-09-23 20:53:27","timestampInMs":1758660807050,"requestPath":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app/favicon.ico","requestMethod":"GET","requestQueryString":"","responseStatusCode":-1,"requestId":"tb6ww-1758660805256-8076a7753c0c","requestUserAgent":"vercel-favicon/1.0","level":"error","environment":"production","branch":"main","vercelCache":"","type":"lambda","function":"/mcp_server.py","host":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app","deploymentDomain":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app","deploymentId":"dpl_AC5hyoygEwFaK8DBZRvDxtmCqmtR","durationMs":-1,"region":"iad1","maxMemoryUsed":-1,"memorySize":-1,"message":"Traceback (most recent call last):\nFile \"/var/task/vc__handler__python.py\", line 38, in <module>\n__vc_spec.loader.exec_module(__vc_module)\nFile \"<frozen importlib._bootstrap_external>\", line 999, in exec_module\nFile \"<frozen importlib._bootstrap>\", line 488, in _call_with_frames_removed\nFile \"/var/task/mcp_server.py\", line 20, in <module>\nfrom mcp.server.auth.provider import TokenVerifier, AccessToken\nImportError: cannot import name 'TokenVerifier' from 'mcp.server.auth.provider' (/var/task/_vendor/mcp/server/auth/provider.py)\nPython process exited with exit status: 1. The logs above can help with debugging the issue.","projectId":"prj_mzlT1PT31kIcvA2XkLZ4NwvwbESq","traceId":"","sessionId":"","invocationId":"01K5W5MGNENVQ2QSTTP15KGVDZ","instanceId":"","concurrency":-1},{"TimeUTC":"2025-09-23 20:53:25","timestampInMs":1758660805294,"requestPath":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app/favicon.ico","requestMethod":"GET","requestQueryString":"","responseStatusCode":500,"requestId":"tb6ww-1758660805256-8076a7753c0c","requestUserAgent":"vercel-favicon/1.0","level":"error","environment":"production","branch":"main","vercelCache":"","type":"lambda","function":"/mcp_server.py","host":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app","deploymentDomain":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app","deploymentId":"dpl_AC5hyoygEwFaK8DBZRvDxtmCqmtR","durationMs":1738,"region":"iad1","maxMemoryUsed":100,"memorySize":2048,"message":"","projectId":"prj_mzlT1PT31kIcvA2XkLZ4NwvwbESq","traceId":"","sessionId":"","invocationId":"01K5W5MGNENVQ2QSTTP15KGVDZ","instanceId":"","concurrency":1},{"TimeUTC":"2025-09-23 20:53:24","timestampInMs":1758660804201,"requestPath":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app/favicon.ico","requestMethod":"GET","requestQueryString":"","responseStatusCode":-1,"requestId":"952gm-1758660802405-13a9bc489a0f","requestUserAgent":"vercel-favicon/1.0","level":"error","environment":"production","branch":"main","vercelCache":"","type":"lambda","function":"/mcp_server.py","host":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app","deploymentDomain":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app","deploymentId":"dpl_AC5hyoygEwFaK8DBZRvDxtmCqmtR","durationMs":-1,"region":"iad1","maxMemoryUsed":-1,"memorySize":-1,"message":"Traceback (most recent call last):\nFile \"/var/task/vc__handler__python.py\", line 38, in <module>\n__vc_spec.loader.exec_module(__vc_module)\nFile \"<frozen importlib._bootstrap_external>\", line 999, in exec_module\nFile \"<frozen importlib._bootstrap>\", line 488, in _call_with_frames_removed\nFile \"/var/task/mcp_server.py\", line 20, in <module>\nfrom mcp.server.auth.provider import TokenVerifier, AccessToken\nImportError: cannot import name 'TokenVerifier' from 'mcp.server.auth.provider' (/var/task/_vendor/mcp/server/auth/provider.py)\nPython process exited with exit status: 1. The logs above can help with debugging the issue.","projectId":"prj_mzlT1PT31kIcvA2XkLZ4NwvwbESq","traceId":"","sessionId":"","invocationId":"01K5W5MDWJSGHZ0YV72MJ70PJG","instanceId":"","concurrency":-1},{"TimeUTC":"2025-09-23 20:53:22","timestampInMs":1758660802450,"requestPath":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app/favicon.ico","requestMethod":"GET","requestQueryString":"","responseStatusCode":500,"requestId":"952gm-1758660802405-13a9bc489a0f","requestUserAgent":"vercel-favicon/1.0","level":"error","environment":"production","branch":"main","vercelCache":"","type":"lambda","function":"/mcp_server.py","host":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app","deploymentDomain":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app","deploymentId":"dpl_AC5hyoygEwFaK8DBZRvDxtmCqmtR","durationMs":1730,"region":"iad1","maxMemoryUsed":100,"memorySize":2048,"message":"","projectId":"prj_mzlT1PT31kIcvA2XkLZ4NwvwbESq","traceId":"","sessionId":"","invocationId":"01K5W5MDWJSGHZ0YV72MJ70PJG","instanceId":"","concurrency":1},{"TimeUTC":"2025-09-23 20:53:22","timestampInMs":1758660802284,"requestPath":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app/","requestMethod":"GET","requestQueryString":"","responseStatusCode":-1,"requestId":"l87g4-1758660800515-9c2dd57771f1","requestUserAgent":"vercel-favicon/1.0","level":"error","environment":"production","branch":"main","vercelCache":"","type":"lambda","function":"/mcp_server.py","host":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app","deploymentDomain":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app","deploymentId":"dpl_AC5hyoygEwFaK8DBZRvDxtmCqmtR","durationMs":-1,"region":"iad1","maxMemoryUsed":-1,"memorySize":-1,"message":"Traceback (most recent call last):\nFile \"/var/task/vc__handler__python.py\", line 38, in <module>\n__vc_spec.loader.exec_module(__vc_module)\nFile \"<frozen importlib._bootstrap_external>\", line 999, in exec_module\nFile \"<frozen importlib._bootstrap>\", line 488, in _call_with_frames_removed\nFile \"/var/task/mcp_server.py\", line 20, in <module>\nfrom mcp.server.auth.provider import TokenVerifier, AccessToken\nImportError: cannot import name 'TokenVerifier' from 'mcp.server.auth.provider' (/var/task/_vendor/mcp/server/auth/provider.py)\nPython process exited with exit status: 1. The logs above can help with debugging the issue.","projectId":"prj_mzlT1PT31kIcvA2XkLZ4NwvwbESq","traceId":"","sessionId":"","invocationId":"01K5W5MC1YJ05B1C5B7W1KX46Z","instanceId":"","concurrency":-1},{"TimeUTC":"2025-09-23 20:53:20","timestampInMs":1758660800573,"requestPath":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app/","requestMethod":"GET","requestQueryString":"","responseStatusCode":500,"requestId":"l87g4-1758660800515-9c2dd57771f1","requestUserAgent":"vercel-favicon/1.0","level":"error","environment":"production","branch":"main","vercelCache":"","type":"lambda","function":"/mcp_server.py","host":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app","deploymentDomain":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app","deploymentId":"dpl_AC5hyoygEwFaK8DBZRvDxtmCqmtR","durationMs":1694,"region":"iad1","maxMemoryUsed":100,"memorySize":2048,"message":"","projectId":"prj_mzlT1PT31kIcvA2XkLZ4NwvwbESq","traceId":"","sessionId":"","invocationId":"01K5W5MC1YJ05B1C5B7W1KX46Z","instanceId":"","concurrency":1},{"TimeUTC":"2025-09-23 20:53:19","timestampInMs":1758660799378,"requestPath":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app/","requestMethod":"GET","requestQueryString":"","responseStatusCode":-1,"requestId":"nj9h9-1758660797609-387f5ed744b9","requestUserAgent":"vercel-favicon/1.0","level":"error","environment":"production","branch":"main","vercelCache":"","type":"lambda","function":"/mcp_server.py","host":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app","deploymentDomain":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app","deploymentId":"dpl_AC5hyoygEwFaK8DBZRvDxtmCqmtR","durationMs":-1,"region":"iad1","maxMemoryUsed":-1,"memorySize":-1,"message":"Traceback (most recent call last):\nFile \"/var/task/vc__handler__python.py\", line 38, in <module>\n__vc_spec.loader.exec_module(__vc_module)\nFile \"<frozen importlib._bootstrap_external>\", line 999, in exec_module\nFile \"<frozen importlib._bootstrap>\", line 488, in _call_with_frames_removed\nFile \"/var/task/mcp_server.py\", line 20, in <module>\nfrom mcp.server.auth.provider import TokenVerifier, AccessToken\nImportError: cannot import name 'TokenVerifier' from 'mcp.server.auth.provider' (/var/task/_vendor/mcp/server/auth/provider.py)\nPython process exited with exit status: 1. The logs above can help with debugging the issue.","projectId":"prj_mzlT1PT31kIcvA2XkLZ4NwvwbESq","traceId":"","sessionId":"","invocationId":"01K5W5M96M3TYNAJVQ6G7HV9VD","instanceId":"","concurrency":-1},{"TimeUTC":"2025-09-23 20:53:17","timestampInMs":1758660797652,"requestPath":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app/","requestMethod":"GET","requestQueryString":"","responseStatusCode":500,"requestId":"nj9h9-1758660797609-387f5ed744b9","requestUserAgent":"vercel-favicon/1.0","level":"error","environment":"production","branch":"main","vercelCache":"","type":"lambda","function":"/mcp_server.py","host":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app","deploymentDomain":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app","deploymentId":"dpl_AC5hyoygEwFaK8DBZRvDxtmCqmtR","durationMs":1702,"region":"iad1","maxMemoryUsed":100,"memorySize":2048,"message":"","projectId":"prj_mzlT1PT31kIcvA2XkLZ4NwvwbESq","traceId":"","sessionId":"","invocationId":"01K5W5M96M3TYNAJVQ6G7HV9VD","instanceId":"","concurrency":1},{"TimeUTC":"2025-09-23 20:53:06","timestampInMs":1758660786578,"requestPath":"puch-mcp-l5d3.vercel.app/favicon.ico","requestMethod":"GET","requestQueryString":"","responseStatusCode":-1,"requestId":"4v6l8-1758660784257-18fd8c9f61f9","requestUserAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/140.0.0.0 Safari/537.36","level":"error","environment":"production","branch":"main","vercelCache":"","type":"lambda","function":"/mcp_server.py","host":"puch-mcp-l5d3.vercel.app","deploymentDomain":"puch-mcp-l5d3.vercel.app","deploymentId":"dpl_AC5hyoygEwFaK8DBZRvDxtmCqmtR","durationMs":-1,"region":"iad1","maxMemoryUsed":-1,"memorySize":-1,"message":"Traceback (most recent call last):\nFile \"/var/task/vc__handler__python.py\", line 38, in <module>\n__vc_spec.loader.exec_module(__vc_module)\nFile \"<frozen importlib._bootstrap_external>\", line 999, in exec_module\nFile \"<frozen importlib._bootstrap>\", line 488, in _call_with_frames_removed\nFile \"/var/task/mcp_server.py\", line 20, in <module>\nfrom mcp.server.auth.provider import TokenVerifier, AccessToken\nImportError: cannot import name 'TokenVerifier' from 'mcp.server.auth.provider' (/var/task/_vendor/mcp/server/auth/provider.py)\nPython process exited with exit status: 1. The logs above can help with debugging the issue.","projectId":"prj_mzlT1PT31kIcvA2XkLZ4NwvwbESq","traceId":"","sessionId":"","invocationId":"01K5W5KWM61E7DEGX1RE3VRKNY","instanceId":"","concurrency":-1},{"TimeUTC":"2025-09-23 20:53:04","timestampInMs":1758660784774,"requestPath":"puch-mcp-l5d3.vercel.app/favicon.ico","requestMethod":"GET","requestQueryString":"","responseStatusCode":500,"requestId":"4v6l8-1758660784257-18fd8c9f61f9","requestUserAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/140.0.0.0 Safari/537.36","level":"error","environment":"production","branch":"main","vercelCache":"","type":"lambda","function":"/mcp_server.py","host":"puch-mcp-l5d3.vercel.app","deploymentDomain":"puch-mcp-l5d3.vercel.app","deploymentId":"dpl_AC5hyoygEwFaK8DBZRvDxtmCqmtR","durationMs":1777,"region":"iad1","maxMemoryUsed":100,"memorySize":2048,"message":"","projectId":"prj_mzlT1PT31kIcvA2XkLZ4NwvwbESq","traceId":"","sessionId":"","invocationId":"01K5W5KWM61E7DEGX1RE3VRKNY","instanceId":"","concurrency":1},{"TimeUTC":"2025-09-23 20:53:03","timestampInMs":1758660783976,"requestPath":"puch-mcp-l5d3.vercel.app/","requestMethod":"GET","requestQueryString":"","responseStatusCode":-1,"requestId":"v7fsm-1758660781375-1c67d659ed06","requestUserAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/140.0.0.0 Safari/537.36","level":"error","environment":"production","branch":"main","vercelCache":"","type":"lambda","function":"/mcp_server.py","host":"puch-mcp-l5d3.vercel.app","deploymentDomain":"puch-mcp-l5d3.vercel.app","deploymentId":"dpl_AC5hyoygEwFaK8DBZRvDxtmCqmtR","durationMs":-1,"region":"iad1","maxMemoryUsed":-1,"memorySize":-1,"message":"Traceback (most recent call last):\nFile \"/var/task/vc__handler__python.py\", line 38, in <module>\n__vc_spec.loader.exec_module(__vc_module)\nFile \"<frozen importlib._bootstrap_external>\", line 999, in exec_module\nFile \"<frozen importlib._bootstrap>\", line 488, in _call_with_frames_removed\nFile \"/var/task/mcp_server.py\", line 20, in <module>\nfrom mcp.server.auth.provider import TokenVerifier, AccessToken\nImportError: cannot import name 'TokenVerifier' from 'mcp.server.auth.provider' (/var/task/_vendor/mcp/server/auth/provider.py)\nPython process exited with exit status: 1. The logs above can help with debugging the issue.","projectId":"prj_mzlT1PT31kIcvA2XkLZ4NwvwbESq","traceId":"","sessionId":"","invocationId":"01K5W5KT39644YZ1KY55FKPMZ2","instanceId":"","concurrency":-1},{"TimeUTC":"2025-09-23 20:53:02","timestampInMs":1758660782185,"requestPath":"puch-mcp-l5d3.vercel.app/","requestMethod":"GET","requestQueryString":"","responseStatusCode":500,"requestId":"v7fsm-1758660781375-1c67d659ed06","requestUserAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/140.0.0.0 Safari/537.36","level":"error","environment":"production","branch":"main","vercelCache":"","type":"lambda","function":"/mcp_server.py","host":"puch-mcp-l5d3.vercel.app","deploymentDomain":"puch-mcp-l5d3.vercel.app","deploymentId":"dpl_AC5hyoygEwFaK8DBZRvDxtmCqmtR","durationMs":1774,"region":"iad1","maxMemoryUsed":100,"memorySize":2048,"message":"","projectId":"prj_mzlT1PT31kIcvA2XkLZ4NwvwbESq","traceId":"","sessionId":"","invocationId":"01K5W5KT39644YZ1KY55FKPMZ2","instanceId":"","concurrency":1},{"TimeUTC":"2025-09-23 20:52:07","timestampInMs":1758660727031,"requestPath":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app/favicon.ico","requestMethod":"GET","requestQueryString":"","responseStatusCode":-1,"requestId":"gnwv4-1758660725188-5f605de89538","requestUserAgent":"vercel-screenshot/1.0","level":"error","environment":"production","branch":"main","vercelCache":"","type":"lambda","function":"/mcp_server.py","host":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app","deploymentDomain":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app","deploymentId":"dpl_AC5hyoygEwFaK8DBZRvDxtmCqmtR","durationMs":-1,"region":"iad1","maxMemoryUsed":-1,"memorySize":-1,"message":"Traceback (most recent call last):\nFile \"/var/task/vc__handler__python.py\", line 38, in <module>\n__vc_spec.loader.exec_module(__vc_module)\nFile \"<frozen importlib._bootstrap_external>\", line 999, in exec_module\nFile \"<frozen importlib._bootstrap>\", line 488, in _call_with_frames_removed\nFile \"/var/task/mcp_server.py\", line 20, in <module>\nfrom mcp.server.auth.provider import TokenVerifier, AccessToken\nImportError: cannot import name 'TokenVerifier' from 'mcp.server.auth.provider' (/var/task/_vendor/mcp/server/auth/provider.py)\nPython process exited with exit status: 1. The logs above can help with debugging the issue.","projectId":"prj_mzlT1PT31kIcvA2XkLZ4NwvwbESq","traceId":"","sessionId":"","invocationId":"01K5W5J2FBFFB1C6362WTNE739","instanceId":"","concurrency":-1},{"TimeUTC":"2025-09-23 20:52:06","timestampInMs":1758660726633,"requestPath":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app/favicon.ico","requestMethod":"GET","requestQueryString":"","responseStatusCode":-1,"requestId":"8wb9p-1758660724727-3e18f1c1ab62","requestUserAgent":"vercel-screenshot/1.0","level":"error","environment":"production","branch":"main","vercelCache":"","type":"lambda","function":"/mcp_server.py","host":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app","deploymentDomain":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app","deploymentId":"dpl_AC5hyoygEwFaK8DBZRvDxtmCqmtR","durationMs":-1,"region":"iad1","maxMemoryUsed":-1,"memorySize":-1,"message":"Traceback (most recent call last):\nFile \"/var/task/vc__handler__python.py\", line 38, in <module>\n__vc_spec.loader.exec_module(__vc_module)\nFile \"<frozen importlib._bootstrap_external>\", line 999, in exec_module\nFile \"<frozen importlib._bootstrap>\", line 488, in _call_with_frames_removed\nFile \"/var/task/mcp_server.py\", line 20, in <module>\nfrom mcp.server.auth.provider import TokenVerifier, AccessToken\nImportError: cannot import name 'TokenVerifier' from 'mcp.server.auth.provider' (/var/task/_vendor/mcp/server/auth/provider.py)\nPython process exited with exit status: 1. The logs above can help with debugging the issue.","projectId":"prj_mzlT1PT31kIcvA2XkLZ4NwvwbESq","traceId":"","sessionId":"","invocationId":"01K5W5J247EGDZFHFJ94FNRN33","instanceId":"","concurrency":-1},{"TimeUTC":"2025-09-23 20:52:05","timestampInMs":1758660725227,"requestPath":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app/favicon.ico","requestMethod":"GET","requestQueryString":"","responseStatusCode":500,"requestId":"gnwv4-1758660725188-5f605de89538","requestUserAgent":"vercel-screenshot/1.0","level":"error","environment":"production","branch":"main","vercelCache":"","type":"lambda","function":"/mcp_server.py","host":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app","deploymentDomain":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app","deploymentId":"dpl_AC5hyoygEwFaK8DBZRvDxtmCqmtR","durationMs":1788,"region":"iad1","maxMemoryUsed":100,"memorySize":2048,"message":"","projectId":"prj_mzlT1PT31kIcvA2XkLZ4NwvwbESq","traceId":"","sessionId":"","invocationId":"01K5W5J2FBFFB1C6362WTNE739","instanceId":"","concurrency":1},{"TimeUTC":"2025-09-23 20:52:04","timestampInMs":1758660724871,"requestPath":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app/favicon.ico","requestMethod":"GET","requestQueryString":"","responseStatusCode":500,"requestId":"8wb9p-1758660724727-3e18f1c1ab62","requestUserAgent":"vercel-screenshot/1.0","level":"error","environment":"production","branch":"main","vercelCache":"","type":"lambda","function":"/mcp_server.py","host":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app","deploymentDomain":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app","deploymentId":"dpl_AC5hyoygEwFaK8DBZRvDxtmCqmtR","durationMs":1745,"region":"iad1","maxMemoryUsed":100,"memorySize":2048,"message":"","projectId":"prj_mzlT1PT31kIcvA2XkLZ4NwvwbESq","traceId":"","sessionId":"","invocationId":"01K5W5J247EGDZFHFJ94FNRN33","instanceId":"","concurrency":1},{"TimeUTC":"2025-09-23 20:52:04","timestampInMs":1758660724811,"requestPath":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app/","requestMethod":"GET","requestQueryString":"","responseStatusCode":-1,"requestId":"4tjh9-1758660722979-189d105a5004","requestUserAgent":"vercel-screenshot/1.0","level":"error","environment":"production","branch":"main","vercelCache":"","type":"lambda","function":"/mcp_server.py","host":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app","deploymentDomain":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app","deploymentId":"dpl_AC5hyoygEwFaK8DBZRvDxtmCqmtR","durationMs":-1,"region":"iad1","maxMemoryUsed":-1,"memorySize":-1,"message":"Traceback (most recent call last):\nFile \"/var/task/vc__handler__python.py\", line 38, in <module>\n__vc_spec.loader.exec_module(__vc_module)\nFile \"<frozen importlib._bootstrap_external>\", line 999, in exec_module\nFile \"<frozen importlib._bootstrap>\", line 488, in _call_with_frames_removed\nFile \"/var/task/mcp_server.py\", line 20, in <module>\nfrom mcp.server.auth.provider import TokenVerifier, AccessToken\nImportError: cannot import name 'TokenVerifier' from 'mcp.server.auth.provider' (/var/task/_vendor/mcp/server/auth/provider.py)\nPython process exited with exit status: 1. The logs above can help with debugging the issue.","projectId":"prj_mzlT1PT31kIcvA2XkLZ4NwvwbESq","traceId":"","sessionId":"","invocationId":"01K5W5J0AB0ZRTCXBDYJPMWYGY","instanceId":"","concurrency":-1},{"TimeUTC":"2025-09-23 20:52:04","timestampInMs":1758660724655,"requestPath":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app/","requestMethod":"GET","requestQueryString":"","responseStatusCode":-1,"requestId":"kq45q-1758660722700-cdf4c4cc5442","requestUserAgent":"vercel-screenshot/1.0","level":"error","environment":"production","branch":"main","vercelCache":"","type":"lambda","function":"/mcp_server.py","host":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app","deploymentDomain":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app","deploymentId":"dpl_AC5hyoygEwFaK8DBZRvDxtmCqmtR","durationMs":-1,"region":"iad1","maxMemoryUsed":-1,"memorySize":-1,"message":"Traceback (most recent call last):\nFile \"/var/task/vc__handler__python.py\", line 38, in <module>\n__vc_spec.loader.exec_module(__vc_module)\nFile \"<frozen importlib._bootstrap_external>\", line 999, in exec_module\nFile \"<frozen importlib._bootstrap>\", line 488, in _call_with_frames_removed\nFile \"/var/task/mcp_server.py\", line 20, in <module>\nfrom mcp.server.auth.provider import TokenVerifier, AccessToken\nImportError: cannot import name 'TokenVerifier' from 'mcp.server.auth.provider' (/var/task/_vendor/mcp/server/auth/provider.py)\nPython process exited with exit status: 1. The logs above can help with debugging the issue.","projectId":"prj_mzlT1PT31kIcvA2XkLZ4NwvwbESq","traceId":"","sessionId":"","invocationId":"01K5W5J05R6A659Z5JBY6YMQJV","instanceId":"","concurrency":-1},{"TimeUTC":"2025-09-23 20:52:03","timestampInMs":1758660723019,"requestPath":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app/","requestMethod":"GET","requestQueryString":"","responseStatusCode":500,"requestId":"4tjh9-1758660722979-189d105a5004","requestUserAgent":"vercel-screenshot/1.0","level":"error","environment":"production","branch":"main","vercelCache":"","type":"lambda","function":"/mcp_server.py","host":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app","deploymentDomain":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app","deploymentId":"dpl_AC5hyoygEwFaK8DBZRvDxtmCqmtR","durationMs":1766,"region":"iad1","maxMemoryUsed":100,"memorySize":2048,"message":"","projectId":"prj_mzlT1PT31kIcvA2XkLZ4NwvwbESq","traceId":"","sessionId":"","invocationId":"01K5W5J0AB0ZRTCXBDYJPMWYGY","instanceId":"","concurrency":1},{"TimeUTC":"2025-09-23 20:52:02","timestampInMs":1758660722871,"requestPath":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app/","requestMethod":"GET","requestQueryString":"","responseStatusCode":500,"requestId":"kq45q-1758660722700-cdf4c4cc5442","requestUserAgent":"vercel-screenshot/1.0","level":"error","environment":"production","branch":"main","vercelCache":"","type":"lambda","function":"/mcp_server.py","host":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app","deploymentDomain":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app","deploymentId":"dpl_AC5hyoygEwFaK8DBZRvDxtmCqmtR","durationMs":1765,"region":"iad1","maxMemoryUsed":100,"memorySize":2048,"message":"","projectId":"prj_mzlT1PT31kIcvA2XkLZ4NwvwbESq","traceId":"","sessionId":"","invocationId":"01K5W5J05R6A659Z5JBY6YMQJV","instanceId":"","concurrency":1},{"TimeUTC":"2025-09-23 20:52:02","timestampInMs":1758660722762,"requestPath":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app/","requestMethod":"GET","requestQueryString":"","responseStatusCode":-1,"requestId":"xlqg2-1758660718779-1b0755241549","requestUserAgent":"vercel-screenshot/1.0","level":"error","environment":"production","branch":"main","vercelCache":"","type":"lambda","function":"/mcp_server.py","host":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app","deploymentDomain":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app","deploymentId":"dpl_AC5hyoygEwFaK8DBZRvDxtmCqmtR","durationMs":-1,"region":"iad1","maxMemoryUsed":-1,"memorySize":-1,"message":"Traceback (most recent call last):\nFile \"/var/task/vc__handler__python.py\", line 38, in <module>\n__vc_spec.loader.exec_module(__vc_module)\nFile \"<frozen importlib._bootstrap_external>\", line 999, in exec_module\nFile \"<frozen importlib._bootstrap>\", line 488, in _call_with_frames_removed\nFile \"/var/task/mcp_server.py\", line 20, in <module>\nfrom mcp.server.auth.provider import TokenVerifier, AccessToken\nImportError: cannot import name 'TokenVerifier' from 'mcp.server.auth.provider' (/var/task/_vendor/mcp/server/auth/provider.py)\nPython process exited with exit status: 1. The logs above can help with debugging the issue.\nTraceback (most recent call last):\nFile \"/var/task/vc__handler__python.py\", line 38, in <module>\n__vc_spec.loader.exec_module(__vc_module)\nFile \"<frozen importlib._bootstrap_external>\", line 999, in exec_module\nFile \"<frozen importlib._bootstrap>\", line 488, in _call_with_frames_removed\nFile \"/var/task/mcp_server.py\", line 20, in <module>\nfrom mcp.server.auth.provider import TokenVerifier, AccessToken\nImportError: cannot import name 'TokenVerifier' from 'mcp.server.auth.provider' (/var/task/_vendor/mcp/server/auth/provider.py)\nPython process exited with exit status: 1. The logs above can help with debugging the issue.","projectId":"prj_mzlT1PT31kIcvA2XkLZ4NwvwbESq","traceId":"","sessionId":"","invocationId":"01K5W5HWACVNSSNQSCRVH1EN43","instanceId":"","concurrency":-1},{"TimeUTC":"2025-09-23 20:52:02","timestampInMs":1758660722715,"requestPath":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app/","requestMethod":"GET","requestQueryString":"","responseStatusCode":-1,"requestId":"npr8f-1758660718635-7ce46a070cd6","requestUserAgent":"vercel-screenshot/1.0","level":"error","environment":"production","branch":"main","vercelCache":"","type":"lambda","function":"/mcp_server.py","host":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app","deploymentDomain":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app","deploymentId":"dpl_AC5hyoygEwFaK8DBZRvDxtmCqmtR","durationMs":-1,"region":"iad1","maxMemoryUsed":-1,"memorySize":-1,"message":"Traceback (most recent call last):\nFile \"/var/task/vc__handler__python.py\", line 38, in <module>\n__vc_spec.loader.exec_module(__vc_module)\nFile \"<frozen importlib._bootstrap_external>\", line 999, in exec_module\nFile \"<frozen importlib._bootstrap>\", line 488, in _call_with_frames_removed\nFile \"/var/task/mcp_server.py\", line 20, in <module>\nfrom mcp.server.auth.provider import TokenVerifier, AccessToken\nImportError: cannot import name 'TokenVerifier' from 'mcp.server.auth.provider' (/var/task/_vendor/mcp/server/auth/provider.py)\nPython process exited with exit status: 1. The logs above can help with debugging the issue.\nTraceback (most recent call last):\nFile \"/var/task/vc__handler__python.py\", line 38, in <module>\n__vc_spec.loader.exec_module(__vc_module)\nFile \"<frozen importlib._bootstrap_external>\", line 999, in exec_module\nFile \"<frozen importlib._bootstrap>\", line 488, in _call_with_frames_removed\nFile \"/var/task/mcp_server.py\", line 20, in <module>\nfrom mcp.server.auth.provider import TokenVerifier, AccessToken\nImportError: cannot import name 'TokenVerifier' from 'mcp.server.auth.provider' (/var/task/_vendor/mcp/server/auth/provider.py)\nPython process exited with exit status: 1. The logs above can help with debugging the issue.","projectId":"prj_mzlT1PT31kIcvA2XkLZ4NwvwbESq","traceId":"","sessionId":"","invocationId":"01K5W5HWACVNSSNQSCRR22REH7","instanceId":"","concurrency":-1},{"TimeUTC":"2025-09-23 20:51:58","timestampInMs":1758660718924,"requestPath":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app/","requestMethod":"GET","requestQueryString":"","responseStatusCode":500,"requestId":"xlqg2-1758660718779-1b0755241549","requestUserAgent":"vercel-screenshot/1.0","level":"error","environment":"production","branch":"main","vercelCache":"","type":"lambda","function":"/mcp_server.py","host":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app","deploymentDomain":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app","deploymentId":"dpl_AC5hyoygEwFaK8DBZRvDxtmCqmtR","durationMs":1758,"region":"iad1","maxMemoryUsed":100,"memorySize":2048,"message":"","projectId":"prj_mzlT1PT31kIcvA2XkLZ4NwvwbESq","traceId":"","sessionId":"","invocationId":"01K5W5HWACVNSSNQSCRVH1EN43","instanceId":"","concurrency":1},{"TimeUTC":"2025-09-23 20:51:58","timestampInMs":1758660718924,"requestPath":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app/","requestMethod":"GET","requestQueryString":"","responseStatusCode":500,"requestId":"npr8f-1758660718635-7ce46a070cd6","requestUserAgent":"vercel-screenshot/1.0","level":"error","environment":"production","branch":"main","vercelCache":"","type":"lambda","function":"/mcp_server.py","host":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app","deploymentDomain":"puch-mcp-l5d3-esn8olg6g-amans-projects-528db5b2.vercel.app","deploymentId":"dpl_AC5hyoygEwFaK8DBZRvDxtmCqmtR","durationMs":1732,"region":"iad1","maxMemoryUsed":100,"memorySize":2048,"message":"","projectId":"prj_mzlT1PT31kIcvA2XkLZ4NwvwbESq","traceId":"","sessionId":"","invocationId":"01K5W5HWACVNSSNQSCRR22REH7","instanceId":"","concurrency":1}]